sstables/sstables_manager.cc

```diff
@@ -152,6 +152,13 @@ sstable_writer_config sstables_manager::configure_writer(sstring origin) const {
     return cfg;
 }
 
+void sstables_manager::reclaim_memory_from_sstable(sstable& sst) {
+    auto memory_reclaimed = sst.reclaim_memory_from_components();
+    _total_memory_reclaimed += memory_reclaimed;
+    _total_reclaimable_memory -= memory_reclaimed;
+    smlogger.info("Reclaimed {} bytes of memory from SSTable components. Total memory reclaimed so far is {} bytes", memory_reclaimed, _total_memory_reclaimed);
```

Comment:
We can make `reclaim_memory_from_components()` return the list of evicted components (in addition to the current return value) and then you can also add that to the log line here, and drop the previous patch.

Reply:
I have updated the log here to print the sstable file name. For now we only reclaim bloom filters so, I have not `reclaim_memory_from_components` to return all the components that are being reclaimed. If/When we add support to reclaim more components, we can update the code to return an exact list.